src/main/java/io/xstefank/wildfly/bot/PullRequestFormatProcessor.java

```diff
@@ -60,6 +61,12 @@ void verifyFormat(@PullRequest.Edited @PullRequest.Opened @PullRequest.Synchroni
         GHPullRequest pullRequest = pullRequestPayload.getPullRequest();
         Map<String, String> errors = new HashMap<>();
 
+        if (Pattern.compile("@%s skip format".formatted(wildFlyBotConfig.githubName()), Pattern.DOTALL | Pattern.CASE_INSENSITIVE | Pattern.LITERAL)
+                .matcher(pullRequest.getBody()).find()) {
+            LOG.info("Skipping format");
```

Comment:
for PR...


keep in mind that we read this in the bot log so how should we associate this log message with anything?